src/Razor/test/testassets/Directory.Build.props

```diff
@@ -2,27 +2,26 @@
   <Import Project="Before.Directory.Build.props" Condition="Exists('Before.Directory.Build.props')" />
 
   <PropertyGroup>
-    <SolutionRoot Condition="'$(SolutionRoot)' == ''">$(MSBuildThisFileDirectory)..\..\</SolutionRoot>
-    <SolutionRoot>$([MSBuild]::EnsureTrailingSlash('$(SolutionRoot)'))</SolutionRoot>
-    <RepositoryRoot>$(SolutionRoot)..\..\</RepositoryRoot>
-
     <!--
       In the case that a user is building a sample directly the MicrosoftNetCompilersToolsetPackagerVersion will not be provided.
       We'll fall back to whatever the current SDK provides in regards to Roslyn's Microsoft.Net.Compilers.Toolset.
     -->
     <BuildingTestAppsIndependently>false</BuildingTestAppsIndependently>
     <BuildingTestAppsIndependently Condition="'$(MicrosoftNetCompilersToolsetPackageVersion)' == ''">true</BuildingTestAppsIndependently>
 
-    <!-- Retarget tests to use the copy of the Sdk from source -->
-    <RazorSdkArtifactsDirectory>$(SolutionRoot)..\..\artifacts\bin\Microsoft.NET.Sdk.Razor\</RazorSdkArtifactsDirectory>
+    <!-- Do not resolve Reference ItemGroup since it has a different semantic meaning in Razor test apps -->
+    <EnableCustomReferenceResolution>false</EnableCustomReferenceResolution>
+    <UseRazorSdk>true</UseRazorSdk>
 
-    <RazorSdkCurrentVersionProps>$(SolutionRoot)src\Microsoft.NET.Sdk.Razor\build\netstandard2.0\Sdk.Razor.CurrentVersion.props</RazorSdkCurrentVersionProps>
-    <RazorSdkCurrentVersionTargets>$(SolutionRoot)src\Microsoft.NET.Sdk.Razor\build\netstandard2.0\Sdk.Razor.CurrentVersion.targets</RazorSdkCurrentVersionTargets>
+    <DefaultNetCoreTargetFramework>net5.0</DefaultNetCoreTargetFramework>
```

Comment:
in aspnetcore-tooling, this is defined in eng\Versions.props, but in this repo, it's in the top level Directory.Build.props. Looks like this file explicitly does not import that file, so I'm just defining it here.

Reply:
This would be really easy to miss when we upgrade - for now can you add a comment to where we define this property in root d.b.t, specifying that this should be updated at the same time?